sockeye/encoder.py

```diff
@@ -265,13 +287,15 @@ def __init__(self, target_layout: str, num_hidden: int, dtype: str = C.DTYPE_FP3
     def encode(self,
                data: mx.sym.Symbol,
                data_length: Optional[mx.sym.Symbol],
-               seq_len: int) -> Tuple[mx.sym.Symbol, mx.sym.Symbol, int]:
+               seq_len: int,
+               att_dict: Optional[Dict[str, mx.sym.Symbol]] = None) -> Tuple[mx.sym.Symbol, mx.sym.Symbol, int]:
         """
         Encodes data given sequence lengths of individual examples and maximum sequence length.
 
         :param data: Input data.
         :param data_length: Vector with sequence lengths.
         :param seq_len: Maximum sequence length.
+        :param att_dict: A dictionary of attention matrices used for visualization.
```

Comment:
Lets use a consistent docstring for this parameter throughout the file.

Reply:
done